File: Baekjoon_Online_Judge/NO_02953/2953.py
# -*- encoding: utf-8 -*-
import sys
r_input = sys.stdin.readline

# ---------------------------- #
# | Created by happiness96   | #
# | Year 2020                | #
# | Month 05                 | #
# | Day 24                   | #
# | 2953 나는 요리사다         | #
# ---------------------------- #


if __name__ == "__main__":
    winner = 0
    winner_score = -1

    for ind in range(1, 6):
        total_score = sum(map(int, r_input().split()))

        if total_score > winner_score:
            winner_score = total_score
            winner = ind
    
    print(winner, winner_score)

File: Baekjoon_Online_Judge/NO_02953/2953.c
#include <stdio.h>
/*
 -------------------------------------- 
 | Created by happiness96             | 
 | Year 2021                          | 
 | Month 01                           | 
 | Day 01                             | 
 | 2953 나는 요리사다                  | 
 -------------------------------------- 
*/

int main(){
    int score[5] = {0, }, max_score = 0, winner = 0;

    for(int i = 0; i < 5; i ++){
        for(int j = 0; j < 4; j ++){
            int s;
            scanf("%d", &s);
            score[i] += s;
        }
    }
    
    for(int i = 0; i < 5; i ++){
        if(score[i] > max_score){
            max_score = score[i];
            winner = i + 1;
        }
    }

    printf("%d %d", winner, max_score);
    return 0;
}